.gitignore

```diff
@@ -6,6 +6,9 @@
 # Keys
 *.jks
 
+# Google Secrets
+android/src/debug/google-services.json
```

Comment:
I believe this is covered [in master](https://github.com/the-blue-alliance/the-blue-alliance-android/blob/master/.gitignore#L13)?